README.md

```diff
@@ -27,7 +27,12 @@ Runs at http://localhost:3000/
 ### Build and run in Docker container
 
 ```bash
+$ # Specify location of API:s
+$ docker build --build-arg BACKEND_API_URL=${API_URL} --build-arg BACKEND_GEOSERVER_URL=${GEOSERVER_URL} --tag=$DOCKER_IMAGE .
```

Comment:
Add an example of API_URL and GEOSERVER_URL?

Reply:
I could add the dev servers url, true.

Reply:
Fixed